listen.js

```diff
@@ -1,47 +1,100 @@
 // const flumeView = require('flumeview-reduce')
 const pull = require('pull-stream')
+const pullMany = require('pull-many')
 const CRUT = require('ssb-crut')
+const { where, and, type, isDecrypted, live: dbLive, toPullStream } = require('ssb-db2/operators')
 
 const { isValid: isAddMember } = require('./spec/group/add-member')
 const { isValid: isExcludeMember } = require('./spec/group/exclude-member')
 
 const poBoxSpec = require('./spec/group/po-box')
-const mockSSB = { backlinks: true, query: true }
-const { isUpdate: isPOBox } = new CRUT(mockSSB, poBoxSpec).spec
 
 module.exports = {
   addMember (ssb) {
     return pull(
-      ssb.messagesByType({ type: 'group/add-member', private: true, live: true }),
+      pullMany([
+        ssb.db.query(
+          where(
+            and(
+              isDecrypted('box2'),
+              type('group/add-member')
+            )
+          ),
+          dbLive({ old: true }),
+          toPullStream()
+        ),
+        pull(
+          ssb.db.reindexed(),
+          pull.filter((msg) => {
+            return msg.value.content.type === 'group/add-member'
+          })
+        )
```

Comment:
```suggestion
        ssb.db.reindexed()
```
we check `isAddMember` below, and that's pretty fast

Reply:
changed now, also in the other places reindexed() is used